packages/stable/src/types.ts

```diff
@@ -672,26 +672,23 @@ export type DatapointsDeleteRequest =
   | (ExternalId & DatapointsDeleteRange);
 
 export interface DatapointAggregates extends DatapointsMetadata {
+  isString: false;
+  isStep: boolean;
```

Comment:
I think we should document `isStep`
you will not get duplicated tsdoc because this property is not common on all union members

Reply:
It's common in two of them, so you get it twice. Will see how the typedoc looks